src/net.cpp

```diff
@@ -2532,7 +2532,7 @@ bool CConnman::Start(CScheduler& scheduler, const Options& connOptions)
     if (fListen && !InitBinds(connOptions)) {
         if (m_client_interface) {
             m_client_interface->ThreadSafeMessageBox(
-                _("Failed to listen on any port. Use -listen=0 if you want this."),
+                _("Failed to listen on any port. Use \u2011listen=0 if you want this."),
```

Comment:
Approach NACK: \u2011 is not a valid character on the commandline, so this will break if someone copy/pastes it.

Reply:
Aha... so replacing " -([a-z]+)=..." with " \u2011{1}={2}"
somewhere in ThreadSafeMessageBox would be OK?

Reply:
No? Users may wish to copy out of GUI dialogs too.

Reply:
Aye, I actually thought about that other case,
but wasn't sure about it.

Reply:
not sure how that would work in Qt, but couldn't you split the text into two strings, and then show each sentence on a new line?

```
_("Failed to listen on any port.")
// insert line break
_("Use -listen=0 if you want this.")
``` 

That's how the screenshot in the OP looks, and it also makes sense semantically

Reply:
> not sure how that would work in Qt, but couldn't you split the text into two strings, and then show each sentence on a new line?
> 
> That's how the screenshot in the OP looks, and it also makes sense semantically

Neat for the window in macOS,
but is there any possibility that this is going to end up
introducing an unnecessary line break in any console output?

In other words, is it somewhere else that actually has to
sort out the passed text better because isn't currently good enough?

Ideally, shouldn't be a responsibility of Bitcoin source code,
but the widget toolkit, or the desktop environment.